addons/binding/org.openhab.binding.homematic/README.md

```diff
@@ -0,0 +1,244 @@
+A virtual datapoint (Switch) to reload all values for all devices, available in channel 0 in GATEWAY-EXTRAS
+
+#### INSTALL_MODE
+A virtual datapoint (Switch) to start the install mode on the gateway, available in channel 0 in GATEWAY-EXTRAS
+
+#### INSTALL_MODE_DURATION
+A virtual datapoint (Integer) to hold the duration for the install mode, available in channel 0 in GATEWAY-EXTRAS (max 300 seconds, default = 60)
+
+#### DELETE_MODE
+A virtual datapoint (Switch) to remove the device from the gateway, available in channel 0 for each device. Deleting a device is only possible if DELETE_DEVICE_MODE is not LOCKED
+
+#### DELETE_DEVICE_MODE
+A virtual datapoint (Enum) to configure the device deletion with DELETE_MODE, available in channel 0 for each device
+* **LOCKED:** (default) device can not be deleted
+* **RESET:** device is reset to factory settings before deleting
+* **FORCE:** device is also deleted if it is not reachable
+* **DEFER:** if the device can not be reached, it is deleted at the next opportunity
+
+**Note:** if you change the value and don't delete the device, the virtual datapoints resets to LOCKED after 30 seconds
+
+#### RELOAD_FROM_GATEWAY
+A virtual datapoint (Switch) to reload all values for the device, available in channel 0 for each device
+
+#### FIRMWARE
+A virtual datapoint (String) to show the firmware version of the device, available in channel 0 for each device
+
+#### BATTERY_TYPE
+A virtual datapoint (String) to show the battery type, available in channel 0 for each battery powered device
+
+#### ON_TIME_AUTOMATIC
+A virtual datapoint (Number) to automatically set the ON_TIME datapoint before the STATE or LEVEL datapoint is sent to the gateway, available for all devices which supports the ON_TIME datapoint  
+This is usefull to automatically turn off the datapoint after the specified time
+
+#### DISPLAY_OPTIONS
+A virtual datapoint (String) to control the display of a 19 button Homematic remote control (HM-RC-19), available on channel 18  
+
+The remote control display is limited to five characters, a longer text is truncated.  
+ 
+You have several additional options to control the display.
+* BEEP _(TONE1, TONE2, TONE3)_ - let the remote control beep
+* BACKLIGHT _(BACKLIGHT_ON, BLINK_SLOW, BLINK_FAST)_ - control the display backlight
+* UNIT _(PERCENT, WATT, CELSIUS, FAHRENHEIT)_ - display one of these units
+* SYMBOL _(BULB, SWITCH, WINDOW, DOOR, BLIND, SCENE, PHONE, BELL, CLOCK, ARROW_UP, ARROW_DOWN)_ - display symbols, multiple symbols possible
+ 
+You can combine any option, they must be separated by a comma. If you specify more than one option for BEEP, BACKLIGHT and UNIT, only the first one is taken into account and all others are ignored. For SYMBOL you can specify multiple options.
+ 
+**Examples:**  
+Assumed you mapped the virtual datapoint to a String item called Display_Options   
+```
+String Display_Options "Display_Options" { channel="homematic:HM-RC-19-B:ccu:KEQ0099999:V_18_DISPLAY_OPTIONS" }
```

Comment:
isn't this an openHAB 1 configuration? I would expect some `channel="..."` here.